src/clone.c

```diff
@@ -140,8 +218,13 @@ SYS_FUNC(clone)
 			printaddr(tcp->u_arg[ARG_CTID]);
 			tprint_arg_name_end();
 		}
+		if (show_namespace && tcp->u_rval >= 0) {
+			tcp->auxstr = get_namespace_auxstr((int)tcp->u_rval, (uint64_t)flags);
```

Comment:
Note that `tcp->u_rval` is in a tracee namespace.  I think it has to be translated into the strace namespace using `translate_pid` before passing on to functions that open files in /proc.

There is no need to cast `flags`.

Reply:
fixed locally.